integration-tests/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -542,6 +542,7 @@ public void lsrTest() throws IOException, TachyonException {
     Assert.assertEquals(expected, mOutput.toString());
     // clear testing username
     System.clearProperty(Constants.SECURITY_LOGIN_USERNAME);
+    MasterContext.reset();
```

Comment:
should we just do MasterContext.reset() in before() for this test (similar to FileSystemMasterTest)